src/infra/src/table/short_urls.rs

```diff
@@ -147,8 +141,7 @@ pub async fn remove(short_id: &str) -> Result<(), errors::Error> {
     Entity::delete_many()
         .filter(Column::ShortId.eq(short_id))
         .exec(client)
-        .await
-        .map_err(|e| Error::DbError(DbError::SeaORMError(e.to_string())))?;
+        .await?;
```

Comment:
This logic moved to [this trait implementation](https://github.com/openobserve/openobserve/pull/5100/files#r1847373471) so it should now happen automatically when `?` converts the error.